### ISSUE TITLE
♿ AVT 3 ♿ Pagination - JAWS user cannot select forward or back

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1591]
User tested with latest version of JAWS 2018, Windows 7, Chrome latest version

Using JAWS the user cannot select forward or back.

**Comment by dakahn** [comment]
@asudoh Is the demonstration of pagination working as intended? Clicking forward and back seems to have no effect -- but it's end of day and I could be missing something.

**Comment by asudoh** [comment]
@dakahn From what I read the code, the only thing the JS code does is telling application that end-user wants to go backward/forward via custom event, and it's application's responsibility to actually go forward/backward. It makes some sense in API perspective because page change involves application logic to fetch the data of new page and rendering it before our pagination component can say "you got to the next page". Probably we need some documentation/example on this. Created #1971 to track that.

**Comment by dakahn** [comment]
Great! We'll keep this open to track any subsequent a11y work.

**Comment by dakahn** [comment]
Closing. This isn't an active issue and I'll open one somewhere to update docs
